grapher/stackedCharts/AbstractStackedChart.tsx

```diff
@@ -36,6 +36,9 @@ export class AbstactStackedChart
             this.selectionArray.selectedEntityNames
         )
 
+        for (const slug of this.yColumnSlugs)
+            table = table.interpolateColumnsLinearly(slug)
```

Comment:
This is applying the transform for each column, which may be an antipattern, idk?
But rewriting `interpolateColumnsLinearly` to work on multiple columns at once seems non-trivial, since the time column can differ for each column slug (I think?).

Reply:
Yeah I think this is fine, I do it for the tolerance interpolation as well for the same reason.

Reply:
IIRC most other plotting libraries always use rows for series, and never columns, and would do a transpose to do that, so might simplify things for us if we do it that way too. (But I haven't double checked)

Reply:
I don't get it, I think the strategy doesn't matter here? Here's how I understand it, in both cases we are interpolating the same way, along time: 

<img width="1115" alt="Screenshot 2020-11-05 at 20 03 06" src="https://user-images.githubusercontent.com/1308115/98290719-f37a3400-1fa1-11eb-9dc8-7c00020599c1.png">

([spreadsheet](https://docs.google.com/spreadsheets/d/1uIwBAB76syIUYaUL6cr-5j2sSSzJVsMdp93Lg7bgBiM/edit?usp=sharing))

Reply:
Awesome! You are totally right. I totally did the algebra wrong.

Reply:
On the original topic, I think doing interpolation one at a time makes sense and is probably the better thing to do even if there was no time unit issue.